src/amo/components/CollectionList/styles.scss

```diff
@@ -20,3 +20,13 @@
     margin: 10px 0 0;
   }
 }
+
+.CollectionList-placeholder {
+  margin: 50px auto;
+  opacity: 0.75;
```

Comment:
I don't see us setting anything else to an `opacity` of `0.75`. Why did you choose to do that? I'm not suggesting it's wrong, just wondering about the motivation to do it. We should also pass this by @pwalm to see what he thinks. He's one of our UX people and generally makes the decisions about how the site should look.

Reply:
i guess when i think "placeholder" text i think lower opacity/lighter text color i guess : ) i can def remove that i just figured styles these were temp styels and were going to be adjusted anyway.

Reply:
Fair enough, that certainly makes sense. Let's see what @pwalm thinks about it.